apps/docs/src/pages/about.tsx

```diff
@@ -0,0 +1,264 @@
+import {
+  faArrowLeft,
+  faCheck,
+  faCrown,
+  faPlus,
+} from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import Layout from '@theme/Layout'
+import TextTransition, { presets } from 'react-text-transition'
+
+const people = [
```

Comment:
It could be cool to maybe add "tags" or something to highlight the current "maintainers" of the tool for people to know who to reach out to? Moreover, we should add all previous contributors as "contributors" - the list of people can be completed through us